Formula/imagemagick.rb

```diff
@@ -69,6 +71,9 @@ def install
       --enable-static
     ]
 
+    args << (build.without? "fontconfig") ? "--without-fontconfig" : "--with-fontconfig"
+    args << (build.without? "freetype") ? "--without-freetype" : "--with-freetype"
+
     if build.without? "modules"
```

Comment:
`depends_on "fontconfig" => :recommended` implies `build.with? "fontconfig"`. Just make my recommended changes locally and see for yourself.